### PR TITLE
don't use pkg for output, use out

### PR DESCRIPTION
**Pull request by jakubgs**
To fix permission errors at end of builds when cleaning.

### CODE REVIEW

File: _assets/ci/Jenkinsfile.android [hunk 1]
```diff
@@ -47,14 +49,15 @@ pipeline {
...
-      dest = "${env.WORKSPACE}/pkg"
+      dest = "${env.WORKSPACE}/out"
       sh "mkdir -p ${dest}"
       /* for easier reuse */
       artifact = "status-go-android-${lib.suffix()}.aar"
     } } }
 
     stage('Setup') { steps { dir(env.STATUS_PATH) {
       sh 'make setup-build'
+      sh 'go get golang.org/x/tools/go/packages'
```
Comment (jakubgs) [inline _assets/ci/Jenkinsfile.android]: A `Makefile` target would probably be neater. No harm in running this install step as part of `setup-build` target right?

**Comment by jakubgs** [comment]
It still fails on this weird error:
```
Building status-go for iOS...
gomobile init
gomobile bind -v -target=ios -ldflags="-s -w" -ldflags ' -X main.buildStamp=2019-08-19.22:33:56 -X github.com/status-im/status-go/params.Version=0.32.0-beta.0 -X github.com/status-im/status-go/params.GitCommit=4fbddc2e -X github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum/metrics.EnabledStr=true' -o build/bin/Statusgo.framework github.com/status-im/status-go/mobile
2019/08/19 23:33:59 go [list -e -json -compiled=true -test=false -export=false -deps=true -find=false -tags ios -- github.com/status-im/status-go/mobile]: exit status 1: go build github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum/crypto/secp256k1: build constraints exclude all Go files in /Users/jenkins/workspace/status-go/platforms/ios/src/github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum/crypto/secp256k1
gomobile: /Users/jenkins/workspace/status-go/platforms/ios/bin/gobind -lang=go,objc -outdir=/var/folders/pc/sgwfk7115_7b8dndw3xgz_0r0000gp/T/gomobile-work-019434551 -tags=ios github.com/status-im/status-go/mobile failed: exit 
```
No idea what `build constraints exclude all Go files` means.

**Comment by jakubgs** [comment]
Seems to be related to these issues:

* https://github.com/golang/go/issues/24068
* https://github.com/golang/go/issues/24433
* https://github.com/golang/go/issues/29074
* https://github.com/golang/go/issues/29268
* https://github.com/golang/go/issues/31661

**Comment by jakubgs** [comment]
This seems to be related to CGO, but not sure how.

**Comment by adambabik** [comment]
@jakubgs I just rerun iOS build and it passed... Magic.

**Review by adambabik — APPROVED** [review]

**Pull request merged**